docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -0,0 +1,188 @@
+{{< /note >}}
+
+
+## Installing with CUDA
+
+ This section of the guide will highlight how to install your GPU driver using [NVIDIA's CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit). The CUDA toolkit is currently fully supported on CentOS 7, OpenSUSE, and other popular distributions.
+
+ Optionally, you can also choose to install the full toolkit.  The full toolkit will provide access to a set of tools that will empower you to create GPU-accelerated applications.
+
+1. Visit the [CUDA Downloads Page](https://developer.nvidia.com/cuda-downloads). The **Select Target Platform** section will show a collection of green buttons that you will select from. When you finish this form, the page will present a download link for the installer. Use these values to complete the form:
+
+    | Prompt | Selection |
+    |--------|-----------|
+    | Operating System | Linux |
+    | Architecture | x86_64 |
+    | Distribution | Your Linode's distribution |
+    | Version | Your distribution's version |
+    | Installer type | runfile (local) |
+
+    An example completed form will look like:
+
+    ![CUDA Downloads Page - Select Target Platform](cuda-downloads-select-target-platform.png "CUDA Downloads Page - Select Target Platform")
+
+1.  A **Download Installer** section will appear below the **Select Target Platform** section. The green **Download** button in this section will link to the installer file. Copy this link to your computer's clipboard:
+
+    ![Copy Download Link](copy-cuda-installer-download-link.png "Right click to copy the download link for the installer")
+
+1.  On your Linode, enter the `wget` command and paste in the download link you copied. This example shows the syntax for the command, but you should make sure to use the download link appropriate for your Linode:
+
+        wget https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.168_418.67_linux.run
+
+1. After wget completes, run your version of the installer script on your Linode to begin the installation process:
+
+        sudo sh cuda_*_linux.run
+
+    {{< note >}}
+The installer will take a few moments to run before generating any output. A full install will take several minutes to fully complete.
+{{< /note >}}
+
+1. Read and accept the License Agreement.
+
+1. Choose to install the CUDA Toolkit in its entirety or partially. To use your GPU, you only need to install the driver. However, the CUDA toolkit has additional functionality that may be useful depending on your use case.
+
+    To only install the driver, uncheck all options directly below the Driver option. This would result in your screen resembling the following:
+
+    ![Cuda Installer](cuda-installer.png "Cuda Installer")
+
+    {{< note >}}
+
+Installation on CentOS and RHEL operating systems will fail following this as the installer requires a reboot to fully remove the default Nouveau driver. Reboot, run the installer again, and your installation will be successful.
```

Comment:
What is the Nouveau driver? Can we include a one-sentence explainer for what that is? Also, is it possible to remove the driver in the Before You Begin section so that it doesn't interrupt the flow here